website/files/models/base.py

```diff
@@ -199,7 +199,7 @@ class StoredFileNode(StoredObject, Commentable):
     checkout = fields.AbstractForeignField('User')
 
     #Tags for a file, currently only used for osfStorage
-    tags = fields.ForeignField('Tag', list=True, index=True)
+    tags = fields.ForeignField('Tag', list=True)
```

Comment:
`raise OperationFailure(details["err"], details["code"])
OperationFailure: Cannot transition from not multi key to multi key in multi statement transaction`

This is the failure when a list is indexed.